site/_posts/2020-10-07-contour_v190.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Announcing Contour v1.9.0
+image: /img/posts/contourauth.png
+excerpt: This blog post covers External Authorization and Cross-Origin Resource Sharing (CORS) Support in Contour v1.9.0.
+author_name: Steve Sloka
+author_avatar: /img/contributors/steve-sloka.png
+categories: [kubernetes]
+# Tag should match author to drive author pages
+tags: ['Contour Team']
+---
+
+Contour continues to add new features to help you better manage ingress operations in a cluster. Our latest feature release, Contour 1.9.0, now includes support for external authorization allowing requests to be validated against an authorization server.  Contour now also supports presenting a client certificate for TLS for validation to backend services as well as moves to v1 Custom Resource Definitions (CRDs).
+
+## External Authorization Support
+Incoming requests to your ingress resources can now be authorized by utilizing Contour’s new external authorization support. Contour now enables the external authorization network filter in Envoy which calls an external authorization service to check if the incoming request is authorized or not. If the request is deemed unauthorized by the network filter then the connection will be closed.
+
+Support for this new feature relies on a new Custom Resource Definition (CRD) named ExtensionService. This new API describes how Envoy should connect to the external authorization server.
+
+![img][1]
+
+### Sequence of Events for External Auth:
+1. Deploy an External Authorization Service to your cluster: This service talks to your Authorization Provider and determines if the request should be authorized or not.
+2. Create an `ExtensionService` CRD: This CRD allows the External Authorization Service created in the previous step to be available so that Contour can configure Envoy with that gRPC endpoint.
+3. Create HTTPProxy resource: The VirtualHost in the ingress object references the ExternalService CRD linking that virtual host to the authorization service. 
+4. On every client request, Envoy sends an authorization check  to the External Auth Service to determine authorization.
+
+## Demo
+Here's a quick video demonstration walking through the setup and configuration of External Authorization with Contour.
+
+[![img][2]][3]
```

Comment:
You should be able to embed the YouTube video through this: https://github.com/projectcontour/contour/blob/main/site/_plugins/youtube.rb

If you replace this with `{% youtube wm_eWO4mZYs %}` it should embed the video.